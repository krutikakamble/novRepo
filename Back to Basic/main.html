<!DOCTYPE html>
<html>
<head>
<title>Template3</title>
<link rel="stylesheet" href="css/hello.css">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
</head>
<body>
<!-- **************************Header sec start***************************************** -->
<header class="header_area">
	<div class="container">
	   	     <section class="top_header_area">
				<h1>BACK TO BASICS</h1>
				<p>Free PSD Website Template</p>
			</section>
			<div class="search-area">
				<form>
				<input type="text" placeholder="Search Our Website...">
				<input type="submit" value="SEARCH">
				</form>
			</div>
			<div class="clear"></div>
			<nav class="main_menu"> 
				<ul>
					<li><a class="active" href="#">HOME</a></li>
					<li><a href="#">STYLE DEMO</a></li>
					<li><a href="#">FULL WIDTH</a></li>
					<li><a href="#">DROPDOWN</a></li>
					<li><a href="#">PORTFOLIO</a></li>
					<li><a href="#">GALLERY</a></li>
				</ul>
			 </nav>
			<div class="logo">
				<div class="logo1">
					<img src="img/icon-linkedin.png"></div>
				<div class="logo2">
					<img src="img/icon-twitter.png"></div>
				<div class="logo3">
					<img src="img/icon-pinterest.png"></div>
				<div class="logo4">
					<img src="img/icon-google-plus.png"></div>
				<div class="logo5">
					<img src="img/icon-rss.png"></div>
				</div>	<div class="clear"></div>
			</div>
</header>
<!-- ****************************Header sec end*****************************************-->
<!-- *************************  Banner sec start*************************************** ***--> 
<article class="container banner-sec">
	<div class="img-sec">
		<img src="img/welcome.jpg">
	</div>
	<div class="banner-text">
		<p>“ Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc non diam erat. In frin gilla massa ut nisi ullamcorper pellentesque „</p>
	</div>
</article>
 <!-- *************************  Banner sec end*************************************** ****-->
 <!-- *************************  Ttile sec start*************************************** ******--> 

 <article class="title-sec1">
	 <div class="container title-sec">
		 <div class="title-text">
			 <h2>YOUR TITLE HERE</h2>
			 <p>Vestibulumaccumsan egestibulum eu justo convallis augue estas aenean elit intesque sed. Facilispede estibulum nulla orna nisl velit elit ac aliquat non tincidunt.</p>
			 <input type="button" value="READ MORE>">
		 </div>
		 <div class="title-text1">
			 <h2>YOUR TITLE HERE</h2>
			 <p>Vestibulumaccumsan egestibulum eu justo convallis augue estas aenean elit intesque sed. Facilispede estibulum nulla orna nisl velit elit ac aliquat non tincidunt.</p>
			 <input type="button" value="READ MORE>">
		 </div>
		 <div class="title-text">
			 <h2>YOUR TITLE HERE</h2>
			 <p>Vestibulumaccumsan egestibulum eu justo convallis augue estas aenean elit intesque sed. Facilispede estibulum nulla orna nisl velit elit ac aliquat non tincidunt.</p>
			  <input type="button" value="READ MORE>">
		 </div>
		 <div class="clear"></div>
	 </div> 
</article>
 
<!-- *************************  Title sec end*************************************** ********--> 
<!-- *************************  About sec start*************************************** ******--> 

<div class="container">
		<article class="service-sec">
			<div class="serive-title">
				<h3>SOME OF OUR SERVICES</h3>
			</div>
			<div class="clear"></div>
			<div class="img-sec1">
				<img src="img/images (2).jpg">
			</div>
			<div class="text">
				<h4>Title Goes Here</h4>
				<p>Vestibulumaccumsan egestibulum eu justo convallis augue estas aenean elit intesque sed. Facilispede estibulum nulla orna nisl velit elit ac aliquat non tincidunt.</p>
			</div>
			<div class="clear"></div>
			<div class="img-sec1">
				<img src="img/images (2).jpg">
			</div>
			<div class="text">
				<h4>Title Goes Here</h4>
				<p>Vestibulumaccumsan egestibulum eu justo convallis augue estas aenean elit intesque sed. Facilispede estibulum nulla orna nisl velit elit ac aliquat non tincidunt.</p>
			</div>
			<div class="clear"></div>
			<div class="img-sec1">
				<img src="img/images (2).jpg">
			</div>
			<div class="text">
				<h4>Title Goes Here</h4>
				<p>Vestibulumaccumsan egestibulum eu justo convallis augue estas aenean elit intesque sed. Facilispede estibulum nulla orna nisl velit elit ac aliquat non tincidunt.</p>
			</div>
			<div class="clear"></div>
			<div class="img-sec1">
				<img src="img/images (2).jpg">
			</div>
			<div class="text">
				<h4>Title Goes Here</h4>
				<p>Vestibulumaccumsan egestibulum eu justo convallis augue estas aenean elit intesque sed. Facilispede estibulum nulla orna nisl velit elit ac aliquat non tincidunt.</p>
			</div>
		</article>

		<article class="service-sec1">
			<div class="serive-title1">
				<h3>A LITTLE ABOUT US</h3>
				<div class="img-sec2">
					<img src="img/images.jpg">
					<p>Justoid nonummy laoreet phasellent penatoque in antesque pellus elis eget tincidunt. Nequatdui laorem justo a non tellus laoreet tincidunt ut vel velit nonummy laoreet phasellent penatoque in antesque pellus elis eget tincidunt.</p>
				</div>
			</div>
			<div class="serive-title2">
				<p>Idenim semper pellente velis felit ac.Justoid nonummy laoreet phasellent penatoque in antesque pellus elis eget tincidunt. Vestibulumaccumsan egestibulum eu justo convallis augue estas aenean elit intesque sed. Facilispede estibulum nulla orna nisl velit elit ac aliquat non tincidunt.</p>  
				<div class="A"><input type="button" value="READ MORE>"></div>
			</div>
			<div class="service-sec2">
				<h3>LATEST TESTIMONIAL</h3>
				<img src="img/Composite-Path.png">
				<h6>Tommy Tanker - CEO</h6>
				<h5>View This Project »</h5>
			</div>
			<p>Justoid nonummy laoreet phasellent penatoque in antesque pellus elis eget tincidunt. Nequatdui laorem justo a non tellus laoreet tincidunt ut vel velit. Idenim semper pellente velis felit ac nullam pretium morbi lacus.</p>
		</article>
		<div class="clear"></div>
</div>
	<!-- *************************  About sec end*************************************** *********--> 
	<!--*************************  Footer sec start*************************************** *******--> 

<Footer class="container Footer-sec">
	<div class="Footer-text">
		<p>Copyright © 2013 Domain Name - All Rights Reserved</p>
		<h6>Template by OS Templates</h6>
	</div>
</Footer>
<!-- *************************  Footer sec end*************************************** ********--> 
</body>
</html>