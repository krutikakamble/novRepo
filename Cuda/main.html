<!DOCTYPE html>
<html>
<head>
<title>Template4</title>
<link rel="stylesheet" href="css/hello.css">
</head>
<body>
<!-- ****************************Header sec start***********************************-->
<Header class="Header-sec">
	<div class="container">
		<div class="Header-text">
			<div class="img-sec">
				<img src="img/logo.png">
			</div>
			 <div class="nav-sec"> 
				<ul>
				<li><a class="active" href="#">HOME</a></li>
				<li><a href="#">ABOUT</a></li>
				<li><a href="#">WORK</a></li>
				<li><a href="#">BLOG </a></li>
				<li><a href="#">CONTACT</a></li>
				</ul>
			</div><div class="clear"></div>                                 
		</div>
		<div class="header-title">
			<h6>Hi there! We are the new kids on the block 
			and we build awesome websites and mobile apps.</h6>
			<button class="button">"WORK WITH US!"</button>
		</div>
	</div>
</Header >
<!-- ****************************Header sec end*************************************** -->
<main class="service-sec">
	<div class="container serive-text">
		<h2 class="underline-small">SERVICES WE PROVIDE</h2>
		<p>We are working with both individuals and businesses from all over the globe 
		to create awesome websites and applications.
		</p>
	</div>
	<div class="container serive-img">
		<div class="img1">
			<img src="img/Flag-icon.png">
			<div class="text-title1">
				<h4>BRANDING</h4>
				<p>Lorem ipsum dolor sit amet,consectetuer adipiscing elit, sed diam nonummy nibh.</p>
			</div>
		</div>
		<div class="img2">
			<img src="img/Shape-18.png">
			<div class="text-title1">
				<h4>DESIGN</h4>
				<p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem  </p>
			</div>
		</div>
		<div class="img3">
			<img src="img/Shape-20.png">
			<div class="text-title1">
				<h4>DEVELOPMENT</h4>
				<p>At vero eos et accusamus et iusto odio dignissimos qui blanditiis praesentium.</p>
			</div>
		</div>
		<div class="img4">
			<img src="img/Shape-28.png">
			<div class="text-title1">
				<h4>ROCKET SCIENCE</h4>
				<p>Et harum quidem rerum est et expedita distinctio. Nam libero tempore.</p>
			</div>
		</div>
	</div>
</main>
<!-- ************************** Team sec start************************************-->
<article  class="team-membr">
	<div class="team-info">
		<h3 class="underline-small1">MEET OUR BEAUTIFUL TEAM</h3>			
		<p>We are a small team of designers and developers, who help brands with big ideas.</p>
	</div>
	<section class="container team1">
		<div class="team-img-info">
			<div class="hello1">
				<img src="img/Anne_Hathaway_in_2017.png">
			</div>
			<div class="team-a">
				<h3>ANNE HATHAWAY</h3>
				<h6>CEO / Marketing Guru</h6>
				<p>Lorem ipsum dolor sit amet,consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna.</p>
			</div>
			<div class="logo">
				<img src="img/Facebook-Icon.png">
				<img src="img/Twitter.png">
				<img src="img/LinkedIn-icon.png">
				<img src="img/Mail.png">
			</div>
		</div>
		<div class="team-img-info1">
			<div class="hello1">
				<img src="img/download.jpg">
			</div>
				<div class="team-a">
					<h3>KATE UPTON</h3>
					<h6>Creative Director</h6>
					<p>Duis aute irure dolor in in voluptate velit esse cillum dolore fugiat nulla pariatur. Excepteur sint occaecat non diam proident.</p>
			    </div>	
					<div class="logo">
						<img src="img/Twitter.png">
						<img src="img/LinkedIn-icon.png">
						<img src="img/Mail.png">
					</div>
				</div>
				<div class="team-img-info2">
					<div class="hello1">
						<img src="img/gettyimages-1077152350-612x612.jpg"></div>
					<div class="team-a">
						<h3>OLIVIA WILDE</h3>
						<h6>Lead Designer</h6>
						<p>Nemo enim ipsam voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem nesciunt.</p>
					</div>	
						<div class="logo">
							<img src="img/Facebook-Icon.png">
							<img src="img/Twitter.png">
							<img src="img/LinkedIn-icon.png">
							<img src="img/Mail.png">
						</div>
				</div>
				<div class="team-img-info3">
					<div class="hello1">
						<img src="img/220px-Ashley_Greene_(42417279271).jpg"></div>
						<div class="team-a">
							<h3>ASHLEY GREENE</h3>
							<h6>SEO / Developer</h6>
							<p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam.</p>
						</div>	
							<div class="logo">
								<img src="img/Facebook-Icon.png">
								<img src="img/Twitter.png">
								<img src="img/Mail.png">
							</div>
				</div>
				<div class="clear"></div> 	
	</section >
</article>
<!-- ************************** Team sec end*************************************-->
<!-- ************************** Skill sec start**********************************-->
<article class="container skill-sec">
	<section class="skill-text">
		<h6 class="underline-small2">WE GOT SKILLS!</h6>
		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
	</section>
	<section class="skill-img">
		<img src="img/Progress.png">
		<img src="img/Progress1.png">
		<img src="img/Progress2.png">
	</section>
		<img src="img/Progress3.png">
</article><div class="clear"></div> 
<!-- ************************** Skill sec end************************************-->
<!-- **************************Porfolio sec start********************************-->
<article class="porfolio-sec">
	<section class="porfolio-text">
		<h4 class="underline-small3">OUR PORTFOLIO</h4>
		<p>Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet consectetur, adipisci velit, sed quia non numquam</p>
	</section>
	<section class="nav">
		<img src="img/All.png">
		<img src="img/WEB.png">
		<img src="img/APPS.png">
	</section>
	<section class="nav1">
		<div class="img5"><img src="img/1.png"><P>Isometric Perspective Mock-Up</p></div>
		<div class="img6"><img src="img/2.png"><P>Time Zone App UI</p></div>
		<div class="img7"><img src="img/3.png"><P>Viro Media Players UI</p></div>
		<div class="img8"><img src="img/4.png"><P>Blog / Magazine Flat UI Kit</p></div>
		<div class="clear"></div> 
	</section>
	<div class="btn">
		<button class="button button2">LOAD MORE PROJECTS</button>
	</div>
</article>
<!-- **************************Porfolio sec end**********************************-->
<!-- **************************About sec start***********************************-->
<article class="aboutus">
	<section class="About-text">
			<h4 class="underline-small4">WHAT POEPLE SAY ABOUT US</h4>
			<p>Our clients love us!
		</p>
		<div class="container about-img">
			<div class="about-img1"><img src="img/images.jpg">
				<div class="about-1">
					<p>“Nullam dapibus blandit orci, viverra gravida dui lobortis eget. Maecenas fringilla urna eu nisl scelerisque.”
					</p><h5>Chanel Iman</h5><h6>CEO of Pinterest</h6>
				</div>
			</div>
			<div class="about-img2"><img src="img/images (1).jpg">
				<div class="about-1">
					<p>"Vivamus luctus urna sed urna ultricies ac tempor dui sagittis. In condimentum facilisis porta.”
					</p><h5>ADRIANA LIMA</h5><h6>Founder of Instagram</h6>
				</div>
			</div><div class="clear"></div> 
				<div class="about-img3"><img src="img/Anne_Hathaway_in_2017.png">
					<div class="about-1"><p>“Vivamus luctus urna sed urna ultricies ac tempor dui sagittis. In condimentum facilisis porta.”
					</p><h5>ANNE HATHAWAY</h5><h6>Lead Designer at Behance</h6>
					</div>
				</div>
			<div class="about-img4"><img src="img/download (1).jpg">
				<div class="about-1"><p>“Phasellus non purus vel arcu tempor commodo. Fusce semper, purus vel luctus molestie, risus sem cursus neque.”
					</p><h5>EMMA STONE</h5><h6>Co-Founder of Shazam</h6>
				</div>
			</div><div class="clear"></div> 
		</div>
	</section>
</article>
<!-- **************************About sec end**************************************-->
<!-- **************************Form sec start*************************************-->
<article class="form-sec">
	<div class="container">
	<div class="form-text">
		<h2 class="underline-small5">GET IN TOUCH</h2>
		<p>1600 Pennsylvania Ave NW, Washington, DC 20500, United States of America. Tel: (202) 456-1111</p>
	</div>
	<form>
		<div class="row">
			<div class="gr1">
				<input type="text" placeholder="YOUR NAME*">
			</div>
			<div class="gr2">
				<input type="text" placeholder="YOUR E-MAIL*">
			</div>
		</div>
		<div class="gr4">
			<textarea class="form-control" placeholder="MASSAGE*"></textarea>
		</div>
		<div class="btn">
			<button class="button button3">SEND MASSAGE</button>
		</div>
	</form>
	</div>
</article>

<!-- **************************Form sec end*******************************************-->
<!-- **************************Fotter sec start***************************************-->
<Footer class="Footer-sec">
	<div class="container">
		<ul type="none">
			<li><a href="#">Facebook</a></li>
			<li><a href="#">Twitter </a></li>
			<li><a href="#">Google+</a></li>
			<li><a href="#">LinkedIn</a></li>
			<li><a href="#">Behance </a></li>
			<li><a href="#">Dribbble</a></li>
			<li><a href="#">GitHub</a></li>
		</ul>
	</div>
</Footer>
<!-- **************************Footer sec end*****************************************-->
</body>
</html>