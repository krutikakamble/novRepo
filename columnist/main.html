<!DOCTYPE html>
<html>
<head>
<title>Template 5</title>
<link rel="stylesheet" href="css/hello.css">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
</head>
<body>
<!-- ****************************Header sec start***********************************-->
<header class="header-tab">
	<div class="container">
			<div class="tab-text">
				<h6>Sign Up | Login | RSS Feeds | Archived News</h6>
			</div>
			<div class="tab-search">
				<input type=search value="Search Our Website...">
			</div>
	
</header>
<div class="container">
	<header class="header-text1">
		<div class="header-text">
			<h1>COLUMNIST</h1>  
			<p>Free PSD Website Template</p>
		</div>
		<div class="header-img">
			<img src="img/panoramic.jpg">
		</div>
		<div class="clear"></div>
	</header>
	<nav class="nav-sec"> 
		<ul>
		<li><a class="active" href="#">HOME</a></li>
		<li><a href="#">STYLE DEMO</a></li>
		<li><a href="#">FULL WIDTH</a></li>
		<li><a href="#">DROPDOWN</a></li>
		<li><a href="#">PORTFOLIO</a></li>
		<li><a href="#">GALLERY</a></li>
		<li><a href="#">LONG TEXT LINK</a></li>
		</ul><div class="clear"></div>
  	</nav>
	
<!-- ****************************Banner sec start*************************************_-->	
<main class="banner-sec">
		<div class="img-sec">
			<img src="img/slider.jpg">
		</div>
		<div class="banner-text">
			<h2>TOP STORY GOES HERE</h2>
			<p>Cursuspenatisaccum ut curabitur nulla tellus tor ames a in curabitur pede. Idet mollisi eros dis orci congue elis et curabitur consequam intesque. Curabiturpisametur in ante.</p>
			<input type="submit" value="READ MORE »">
		</div><div class="clear"></div>
	</div>
</main>
<!-- ****************************Banner sec end*************************************_-->	
<!-- ****************************Header sec end************************************* -->
<!-- ****************************Info sec start*************************************_-->
<div class="container">
	<article class="main">
		<section class="info-sec">
			<div class="info-text">
				<h5>Nulla facilisi. Ut fringilla. Suspendisse potenti. Nunc feugiat mi a tellus consequat imperdiet.</h5>
				<p>6th April 2000</p>
				<h6>Read The Full Story »</h6>
			</div>
			<div class="info-img1">
				<img src="img/images (3).jpg">
			</div>
			<div class="info-sec">
				<div class="info-text">
					<h5>Nulla facilisi. Ut fringilla. Suspendisse potenti. Nunc feugiat mi a tellus consequat imperdiet.</h5>
					<p>6th April 2000</p>
					<h6>Read The Full Story »</h6>
				</div>
				<div class="info-img2">
					<img src="img/download (2).jpg">
				</div>
			</div>
		</section>
		<section class="info-sec1">
			<div class="info-text">
				<h5>Nulla facilisi. Ut fringilla. Suspendisse potenti. Nunc feugiat mi a tellus consequat imperdiet.</h5>
				<p>6th April 2000</p>
				<h6>Read The Full Story »</h6>
			</div>
			<div class="info-img2">
				<img src="img/download (2).jpg">
			</div>
			<div class="info-sec">
				<div class="info-text">
					<h5>Nulla facilisi. Ut fringilla. Suspendisse potenti. Nunc feugiat mi a tellus consequat imperdiet.</h5>
					<p>6th April 2000</p>
					<h6>Read The Full Story »</h6>
				</div>
				<div class="info-img1">
					<img src="img/images (3).jpg">
				</div>
			</div>
		</section>
		<section class="info-sec0">
			<div class="info-sec-1">
			<div class="info-text">
				<h5>Nulla facilisi. Ut fringilla. Suspendisse potenti. Nunc feugiat mi a tellus consequat imperdiet.</h5>
				<p>6th April 2000</p>
				<h6>Read The Full Story »</h6>
			</div>
			<div class="info-img3">
				<img src="img/download.jpg">
			</div></div>
			<div class="info-sec-2">
			<div class="info-text">
				<h5>Nulla facilisi. Ut fringilla. Suspendisse potenti. Nunc feugiat mi a tellus consequat imperdiet.</h5>
				<p>6th April 2000</p>
				<h6>Read The Full Story »</h6>
			</div>
			<div class="info-img4">
				<img src="img/images (2).jpg">
			</div></div>
		</section>
		<div class="clear"></div>
	</article>
</div>
<!-- ****************************Info sec end*************************************-->
<!-- ****************************Footer sec start**********************************-->
<footer class="container footer-sec">
	<div class="footer-text">
		<h6>Lacus interdum</h6>
		<p>Lorem ipsum dolor
		Suspendisse in neque
		Praesent et eros
		Lorem ipsum dolor
		Suspendisse in neque
		Praesent et eros</p>
	</div>
	<div class="footer-text1">
		<h6>Lacus interdum</h6>
		<p>Lorem ipsum dolor
		Suspendisse in neque
		Praesent et eros
		Lorem ipsum dolor
		Suspendisse in neque
		Praesent et eros</p>
	</div>
	<div class="footer-text1">
		<h6>Lacus interdum</h6>
		<p>Lorem ipsum dolor
		Suspendisse in neque
		Praesent et eros
		Lorem ipsum dolor
		Suspendisse in neque
		Praesent et eros</p>
	</div>
	<div class="footer-text1">
		<h6>Lacus interdum</h6>
		<p>Lorem ipsum dolor
		Suspendisse in neque
		Praesent et eros
		Lorem ipsum dolor
		Suspendisse in neque
		Praesent et eros</p>
	</div>
	<div class="footer-text">
		<h6>Lacus interdum</h6>
		<p>Lorem ipsum dolor
		Suspendisse in neque
		Praesent et eros
		Lorem ipsum dolor
		Suspendisse in neque
		Praesent et eros</p>
	</div><div class="clear"></div>
</footer>
<!-- ****************************Footer sec end************************************-->
<!-- ****************************Copyright sec start**********************************-->
<nav class="copyright-sec">
	<div class="container copyright-text">
		<p>Copyright © 2013 Domain Name - All Rights Reserved</p>
		<h6>Template by OS Templates</h6>
	</div>
</nav>
<!-- ****************************Copyright sec end************************************-->
</body>
</html>
