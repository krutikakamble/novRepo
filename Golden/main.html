<!DOCTYPE html>
<html>
<head>
<title>Template2</title>
<link rel="stylesheet" href="css/hello.css">
</head>
<body>
<!-- ****************************Header Start**************************************-->
<!-- *************************************************************************** -->
<div class="header">
	<div>
		<img src="img/img11.png">
	</div>
	<div class="container nav">
			<div class="nav-logo">
				<h2>Golden</h2>
			</div>
		<div class="nav-links"> 
			<ul>
				<li><a href="#">HOME</a></li>
				<li><a href="#">SERVICES</a></li>
				<li><a href="#">PORTFOLIO</a></li>
				<li><a href="#">ABOUT</a></li>
				<li><a href="#">CONTACT</a></li>
			</ul>
		</div> 
			<div class="clear"></div> 
	</div>
</div>
	<div class="header-content">
		<h3>Welcome To Our Studio!</h3>
		<h2>IT’S NICE TO MEET YOU</h2>
		<a>TELL ME MORE</a>
	</div>

<!-- ****************************Header End**************************************-->
<!-- ************************** Services sec start***********************************-->
<div class="container">
	<div class="services-heading">
		<h5>SERVICES</h5>
		<p>Proin iaculis purus consequat sem cure. 
	</div>
	<section class="services">
		<div class="section">
				<img src="img/Icon-Basket.png">
				<h4>E-Commerce</h4>
				<p>Proin iaculis purus consequat sem cure 
				digni ssim. Donec porttitora entum suscipit 
				aenean rhoncus posuere odio in tincidunt.</p>
		</div>
		<div class="section">
				<img src="img/Icon-Laptop.png">
				<h4>Responsive Web</h4>
				<p>Proin iaculis purus consequat sem cure 
				digni ssim. Donec porttitora entum suscipit 
				aenean rhoncus posuere odio in tincidunt.</p>
		</div>
		<div class="section">
				<img src="img/Icon-Locked.png">
				<h4>Web Security</h4>
				<p>Proin iaculis purus consequat sem cure 
				digni ssim. Donec porttitora entum suscipit 
				aenean rhoncus posuere odio in tinciduntcontainer1.</p>
		</div><div class="clear"></div> 
	</section>
</div>		
<!-- ************************** Services sec end************************************-->
<!-- ************************** Portfolio sec start***********************************-->
<article class="portfolio">
	<div class="container ">
	<div class="portfolio-heading">
		<h2>OUR PORTFOLIO</h2>
		<p>Proin iaculis purus consequat sem cure. </p>
	</div>
	<div class="coimg-gallary">
		<div class="img-sec">
				<div>
					<img src="img/Untitled-1.png">
				</div>
				<div class="img-info">
					<h4>Ebony & Ivory</h4>
					<h6>Branding</h6>
				</div>
		</div>
		<div class="img-sec">
				<div class="title">
					<img src="img/Untitled-2.png">
				</div>
				<div class="img-info">
					<h4>Smart Stationary</h4>
					<h6>Print Design</h6>
				</div>
		</div>
		<div class="img-sec">
				<div>
					<img src="img/Untitled-3.png">
				</div>
				<div class="img-info">
					<h4>Clever Poster</h4>
					<h6>Print Design</h6>
				</div>
		</div><div class="clear"></div> 
		<div class="img-sec">
				<div>
					<img src="img/Untitled-4.png">
				</div>
				<div class="img-info">
					<h4>Vinyl Record</h4>
					<h6>Product Mock-Up</h6>
				</div>
		</div>
		<div class="img-sec">
				<div>
					<img src="img/Untitled-5.png">
				</div>
				<div class="img-info">
					<h5>Treehouse Template</h5>
					<h6>Web Design</h6>
				</div>
		</div>
		<div class="img-sec">
				<div>
					<img src="img/Untitled-6.png">
				</div>
				<div class="img-info">
					<h4>Burned Logo</h4>
					<h6>Branding</h6>
				</div>
		</div>
	</div>
</div>
</article>
<!-- ************************** Portfolio sec end***********************************-->
<!-- ************************** About sec start************************************-->
<article class="about">
		<div class= "about-heading">
			<h2>ABOUT US</h2>
			<p>Proin iaculis purus consequat sem cure. </p>
		</div>
		<section class="container about-us-gallary">
				<div class="about-img-pic">
						<div class="about-info">
							<h3>JULY 2010 </h3><h3>Our Humble Beginnings</h3>
							<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.</p>
						</div>
						<div class="about-img">
							<img src="img/Image10.png">
						</div>
						<div class="clear"></div> 
				</div>
				<div class="about-img-pic1">
						<div class="about-img1">
							<img src="img/Image11.png">
						</div>
						<div class="about-info1">
							<h3>JANUARY 2011</h3><h3> Facing Startup Battles </h3>
							<p>Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt. Proin 
							iaculis purus consequat sem cure digni 
							ssim. Donec porttitora entum suscipit.</p>
							</div>
						<div class="clear"></div> 
				</div>
				<div class="about-img-pic">
						<div class="about-info">
								<h3>DECEMBER 2012</h3><h3> Enter The Dark Days	</h3>
								<p>Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt. Proin 
								iaculis purus consequat sem cure digni 
								ssim. Donec porttitora entum suscipit.</p>
						</div>
						<div class="about-img">
							<img src="img/Image10.png">
						</div>
						<div class="clear"></div> 
				</div>
				<div class="about-img-pic1">
						<div class="about-img1">
							<img src="img/Image13.png">
						</div>
						<div class="about-info1">
							<h3>FEBRUARY 2014</h3><h3> Our Triumph</h3>
							<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.</p>
						</div>
						<div class="clear"></div> 
				</div>
				<div class="about-img2">
						<img src="img/Our-Story-Continues-..png">
				</div>	
					<div class="clear"></div> 
				</section>
</article>	
<!-- ************************** About sec end************************************-->
<!-- ************************** Team sec start************************************-->
<article class="team-membr">
		<div class="container team-info">
				<h3>OUR AMAZING TEAM</h3>			
				<p>Proin iaculis purus consequat sem cure. </p>
		</div>
		<div class="container team1">
				<div class="team-img-info">
							<div class="hello1">
							<img src="img/KK.jpg"></div>
							<div class="team-a">
							<h6>Kimberly Thompson</h6>
							<p>Marketer</p></div>
							<div class="logo">
							<img src="img/Twitter.png">
							<img src="img/Facebook.png">
							<img src="img/Pinterest.png">
							</div>
				</div>
				<div class="team-img-info">
							<div class="hello1">
							<img src="img/RP.jpg"></div>
							<div class="team-a">
							<h6>Rico Massimo</h6>
							<p>Coder</p></div>
							<div class="logo">
							<img src="img/Twitter.png">
							<img src="img/Facebook.png">
							<img src="img/Google+.png">
							</div>
				</div>
				<div class="team-img-info">
							<div class="hello1">
							<img src="img/MM.jpg"></div>
							<div class="team-a">
							<h6>Uku Mason</h6>
							<p>Graphic Designer</p></div>
							<div class="logo">
							<img src="img/Twitter.png">
							<img src="img/Pinterest.png">
							</div>
				</div><div class="clear"></div> 
				<div class="team-info2">
							<p>Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit  
                             aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
				</div>
</div>
</article>
<!-- ************************** Team sec end************************************-->
<!-- ************************** mid sec end************************************-->
<div class="container "> 
	<section class="block-img">
			<img src="img/Envato.png">
			<img src="img/WordPress.png">
			<img src="img/Tuts+.png">
			<img src="img/Microlancer-copy.png">
	</section></div>
<!-- ************************** mid sec end************************************-->
<!-- ************************** Map sec end************************************-->
 <form class="map-gallary"> 
		<div class="map-gal">
			<img src="img/map.png">
		</div>
		<div class="map-content">
				<h3>CONTACT US</h3>
				<p>Proin iaculis purus consequat sem cure. </p>
			<div class="row">
				<div class="col">
					<div class="form-gr1">
						<input type="text" placeholder="YOUR NAME*">
					</div>
					<div class="form-gr2">
						<input type="text" placeholder="YOUR E-MAIL*">
					</div>
					<div class="form-gr3">
						<input type="text" placeholder="SUBJECT">
					</div>
				</div>
				<div class="col2">
					<div class="form-gr4">
						<textarea class="form-control" placeholder="MASSAGE*"></textarea>
					</div>
				</div><div class="clear"></div> </div>
				<button class="button">SEND MASSAGE</button>
			</div>
		</form>
		</div>
</form>

<!-- ************************** Map sec end************************************-->
<footer class="container footer">
	<div class="footer-text">
		<p>© Copyright 2014 FreebiesXpress.com</p>
	</div>
	<div class="footer-img">
		<img src="img/Twitter1.png">
		<img src="img/Facebook1.png">
		<img src="img/Pinterest1.png">
		<img src="img/Google+1.png">
	</div>
</footer>
</body>
</html>